git_machete/exceptions.py

```diff
@@ -14,14 +14,15 @@ def __init__(self) -> None:
         pass
 
 
-class UnprocessableEntityHTTPError(Exception):
+class UnprocessableEntityHTTPError(MacheteException):
     """This exception is raised when Github API returns HTTP status code 422 - Unprocessable Entity.
     Such a situation occurs when trying to do something not allowed by GitHub,
     e.g. assigning someone from outside organization as a reviewer
     or creating a pull request for a branch that already has a PR.
     """
     def __init__(self, msg: str) -> None:
-        self.msg: str = msg + "\nPlease open an issue regarding this topic under link: https://github.com/VirtusLab/git-machete/issues/new"
+        self.msg: str = 'UnprocessableEntityHTTPError: ' + msg + \
+                        "\nPlease open an issue regarding this topic under link: https://github.com/VirtusLab/git-machete/issues/new"
```

Comment:
Hmmm I don't think the user should really open a new issue against git-machete just when 422 happens due 🤔